Vagrantfile

```diff
@@ -301,6 +301,7 @@ Vagrant.configure('2') do |config|
 
   config.vm.define :s_standalone do |cfg|
     default_omnibus config
+    config.vm.box = 'bento/centos-7.6'
```

Comment:
Thoughts on just changing line 78 to centos-7.6 for all the boxes instead of just overriding it for these two?

Reply:
+1

Reply:
Done in the latest commit.